resources/views/v2/staff/contacts/email/form.blade.php

```diff
@@ -0,0 +1,78 @@
+@extends('v2.layouts.no_drawer')
+
+@section('title', (empty($contact_email) ? 'メールアドレス追加' : $contact_email->name). ' - お問い合わせ')
+
+@section('navbar')
+    <app-nav-bar-back inverse href="{{ route('staff.contacts.email.index') }}">
+        メールアドレス一覧
+    </app-nav-bar-back>
+@endsection
+
+@section('content')
+    <app-container medium>
+        <form
+            action="{{ isset($contact_email)
+                ? route('staff.contacts.email.update', ['contact_email' => $contact_email])
+                : route('staff.contacts.email.create') }}"
+            method="post">
+        @csrf
+        @method(isset($contact_email) ? 'patch' : 'post')
+            <list-view>
+                <template v-slot:title>
+                    {{ isset($contact_email) ? $contact_email->name : 'メールアドレス追加'}}
+                    @isset($contact_email)
+                        <small><a href="{{ route('staff.contacts.email.delete', $contact_email) }}">削除</a></small>
+                    @endisset
+                    - お問い合わせ
+                </template>
+                <list-view-form-group label-for="name">
+                    <template v-slot:label>
+                        項目名
+                    </template>
+                    <template v-slot:description>
+                        この名前がお問い合わせの宛先に表示されます
```

Comment:
もうちょっとわかりやすい日本語にしたいです！
…と思ったんだけど、いい感じの日本語が僕も思い浮かばない、、ここ難しいよね、、

```suggestion
                        項目名は「お問い合わせ」内容を選ぶ選択肢としてユーザーに表示されます
```